### ISSUE TITLE
Improve interactive element target sizes for accessibility

### ISSUE THREAD

**Issue by spli02** [fortanix/baklava#485]
Some interactive elements do not meet the minimum target size requirement defined in WCAG 2.2 (https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).
To improve accessibility, the target size of the interactive elements should be increased to at least 24×24 CSS pixels.

During my work on the internal app, I found the following issues:

| Component                       | Current Target Size                                         |
|---------------------------------|------------------------------------------------------------|
| `IconButton` (e.g., close button on dialogs, remove button of Tag)                     | 17×17  |
| `Checkbox` of `DataTable`            | 15×15                                                      |
| Sort icon of `DataTable`           | 14×13                                                      |
| Page control button of `DataTable` | 28×17                                                      |
| Pagination button of `DataTable`   | 13×13                                                      |
| `Switch`                          | 40×20                                                      |
| `InputField` / `Select`            | Height: 20px                                               |
| `Radio`                           | 18×18                                                      |

We should also review other components to ensure the minimum target size.
